sql/5_0_2-to-6_0_0_upgrade.sql

```diff
@@ -2136,6 +2136,22 @@ ALTER TABLE `form_bronchitis` MODIFY `diagnosis3_bronchitis_form` text;
 ALTER TABLE `form_bronchitis` MODIFY `diagnosis4_bronchitis_form` text;
 #EndIf
 
+#IfColumn patient_data care_team
+ALTER TABLE `patient_data` CHANGE `care_team` `care_team_provider` text;
+#EndIf
+
+#IfMissingColumn patient_data care_team_facility
+ALTER TABLE `patient_data` ADD COLUMN `care_team_facility` text;
+#EndIf
+
+#IfRow2D layout_options form_id DEM field_id care_team
+UPDATE `layout_options` SET field_id='care_team_provider', seq=22, group_id=3, title='Care Team (Provider)', data_type=45 WHERE form_id='DEM' AND field_id='care_team';
```

Comment:
In this case, we need to collect the current seq id in the group this lies in (since these fields are very customizable), and then add 1 to it. Should look something like this:
```
SET @group_name = (SELECT group_name FROM layout_options WHERE field_id='care_team' AND form_id='DEM');
SET @seq = (SELECT MAX(seq) FROM layout_options WHERE group_name = @group_name AND form_id='DEM');
UPDATE `layout_options` SET field_id='care_team_provider', seq=@seq+1, title='Care Team (Provider)', data_type=45 WHERE form_id='DEM' AND field_id='care_team';
```

Reply:
Thanks for sharing. Didn't know those capablities of sql.